src/components/utils/jwt.js

```diff
@@ -11,15 +11,11 @@ export const jwtExpired = (token) => {
   return currentSeconds() > expTime
 }
 
-export const fetchUrlsFromJWT = (token, onInvalidJWT) => {
-  let urls = null
+export const fetchUrlsFromJWT = (token) => {
+  let urls = {}
   try {
     const jwt = parseJwt(token)
     urls = jwt.urls
-  }
-  catch (err) {
-    console.error('Invalid token:', err.message)
-    onInvalidJWT()
-  }
+  } catch {}
```

Comment:
I still think that having `try...catch` like such is risky, as in case of some error happening within `try` statement you won't get anything in output - you silence the error.

Reply:
Agreed.

Reply:
Cool, I remove the noise and brought back the console.error